SRTNet.h

```diff
@@ -174,17 +174,18 @@ class SRTNet {
     SRTNet &operator=(SRTNet const &) = delete;  // Copy assign
     SRTNet &operator=(SRTNet &&) = delete;      // Move assign
 
-
-private:
-
-    //Internal variables and methods
-
     enum class Mode {
         unknown,
         server,
         client
     };
 
+    Mode getCurrentMode();
```

Comment:
```suggestion
    Mode getCurrentMode() const;
```
+ missing doxygen comment
Also move this to under the last member function (getConnectedServer)